src/main/java/models/Music.java

```diff
@@ -1,12 +1,16 @@
 package models;
+import java.time.LocalDate;
+import java.util.List;
+import exception.YearException;
 
 public class Music {
 
     private int id;
     private String genre;
     private String format;
     private  int year;
-    private String artist;
```

Comment:
I think it's better to keep artist as a string. Because the artist of the music should be the one. For instance: Maroon5 is an artist of the album. You can take a look on our target website.

Reply:
We have from the test overview that artists can not be more then 3! So that is mean artist is not a String, that is why i changed it

Reply:
Yeah, then we put the misconception against those issue... revert it back to string to make it correct. Don't worry about the number of test cases.

Reply:
Okey 
I will return it back to string then.

Reply:
Thanks

Reply:
Can you accept the request for now and then i will work in it again.